commons/src/main/java/fr/free/nrw/commons/MediaWikiImageView.java

```diff
@@ -151,7 +151,7 @@ private void loadImageIfNecessary(final boolean isInLayoutPass, final boolean tr
         // from the network.
         ImageContainer newContainer = mImageLoader.get(mUrl,
                 new ImageListener() {
-                    @Override
```

Comment:
We definitely want to keep all @Overrides. Using an @Override on a method that's not actually overridden will cause a compiler error. This is what we want if we think we're overriding something we're actually not. Static code analysis like this will save time in debugging where no tests exist.